src/slices-and-lifetimes/lifetime-elision.md

```diff
@@ -61,7 +61,7 @@ references in its arguments that requires explicit annotation.
 Try adjusting the signature to "lie" about the lifetimes returned:
 
 ```rust,ignore
```

Comment:
We could perhaps take out the `ignore` part here — and fill the body with a `todo!()`?

That should have prevented this kind of tiny typo from occurring. Alternatively, we could put the Rust code in a separate file and include just the line needed to demonstrate the point.

Just some food for thought in case @IP1llar has some free time and want to send us more PRs :smile: